accname/name/comp_host_language_label.html

```diff
@@ -140,27 +140,8 @@ <h2>HTML table/caption</h2>
 </table>
 
 
-<h2>SVG title</h2>
-<svg viewbox="0 0 300 100">
-  <circle cx="26" cy="26" r="25" data-expectedlabel="circle label" data-testname="svg: circle > title" class="ex">
-    <title>circle label</title>
-  </circle>
-
-  <rect x="60" y="1" width="50" height="50" data-expectedlabel="rect label" data-testname="svg: rect > title" class="ex">
-    <title>rect label</title>
-  </rect>
-  <polygon points="100,100 150,25 150,75 200,0" fill="none" stroke="black" data-expectedlabel="polygon label" data-testname="svg: polygon > title" class="ex">
-    <title>polygon label</title>
-  </polygon>
-</svg><br>
-<svg viewbox="0 0 200 90">
-  <g fill="white" stroke="green" stroke-width="5" data-expectedlabel="group label" data-testname="svg: g > title" class="ex">
-    <title>group label</title>
-    <circle cx="40" cy="40" r="25" />
-    <circle cx="60" cy="60" r="25" />
-  </g>
-</svg><br>
-<!-- todo: more SVG tests -->
+<!-- SVG: -> /svg-aam/name/ -->
```

Comment:
Reviewers note: these removals were moved entirely to the new file below, and extended beyond the `title` element to include the `xlink:title` tests with `[xlink:title][href]` and `[xlink:title][xlink:href]:not([href])`